src/components/expanded-state/swap-settings/MaxToleranceInput.tsx

```diff
@@ -105,21 +110,35 @@ export const MaxToleranceInput: React.FC<{
     [updateSwapSlippage, setSlippageValue]
   );
 
-  const hasPriceImpact = isSeverePriceImpact || isHighPriceImpact;
+  const hasPriceImpact = isHighPriceImpact || isSeverePriceImpact;
+
+  const openExplainer = () => {
+    android && Keyboard.dismiss();
```

Comment:
i think you also need to add a function to unfocus the slippage input here because i ended up in a weird state when going to swap settings and tapping on an explainer icon

![image](https://user-images.githubusercontent.com/6843656/175683809-8eab9af2-f29f-4a47-ba01-e75956d96cec.png)

Reply:
either that or delay the keyboard dismiss by 150ms so it's fired after the bottom sheet opens.

steps to repro:

1. go to swap settings
2. quickly tap on one of the explainer items
3. bottom sheet slides up
4. keyboard slides up over the bottom sheet